File: old/Python/중간고사 대비/3-1.py
p1 = int(input())
p2 = int(input())
p3 = int(input())

if p1 >= p2 and p1 >= p3:
    print('Max = %d'%(p1))
elif p2 >= p1 and p2 >= p3:
    print('Max = %d'%(p2))
elif p3 >= p1 and p3 >= p2:
    print('Max = %d'%(p3))

    

File: C/과제/과제3/3-1.c
#include <stdio.h>

int main() {
    int p, c, cnt = 1, max = 0;

    scanf("%d", &p);

    while (1) {
        scanf("%d", &c);

        if (c == 0) {
            if (cnt > max) max = cnt;
            break;
        }

        if (p*c > 0) cnt++;
        else {
            if (cnt > max) max = cnt;
            cnt = 1;
        }

        p = c;
    }

    printf("%d", max);

    return 0;
}